.github/workflows/python-app.yml

```diff
@@ -53,17 +53,25 @@ jobs:
       run: |
         cd bot
         python -m twitter_api.tweet_plot
-        python -m twitter_api.sync_last_seen_id
         git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
         git config --local user.name "github-actions[bot]"
         git pull origin main
-        git commit -m "Update config, data and last_seen_id - ci skip" -a
+        git commit -m "Update config, data - ci skip" -a
       env:
         ACCESS_KEY: ${{ secrets.ACCESS_KEY }}
         ACCESS_SECRET: ${{ secrets.ACCESS_SECRET }}
         CONSUMER_KEY: ${{ secrets.CONSUMER_KEY }}
         CONSUMER_SECRET: ${{ secrets.CONSUMER_SECRET }}
 
+    - name: Sync last seen ID
```

Comment:
This was added to sync the `last_seen_id` on every push because `Heroku` builds and deploys the bot every time the CI passes on the main branch and not just only on scheduled CIs.